tls/s2n_server_extensions.c

```diff
@@ -88,19 +92,30 @@ int s2n_server_extensions_send(struct s2n_connection *conn, struct s2n_stuffer *
         total_size += s2n_extensions_server_key_share_send_size(conn);
     }
 
+    return total_size;
+}
+
+int s2n_server_extensions_send(struct s2n_connection *conn, struct s2n_stuffer *out)
+{
+    int total_size = s2n_server_extensions_send_size(conn);
+
+    GUARD(total_size);
```

Comment:
i think you want:

`GUARD(total_size <= 65535);`

Reply:
no, this is guarding for negative returns